mgmt_cli_test.py

```diff
@@ -142,15 +142,13 @@ def test_backup_feature(self):
         with self.subTest('Test Backup Rate Limit'):
             self.test_backup_rate_limit()
 
-    def update_cred_file(self):
+    def update_config_file(self):
         # FIXME: add to the nodes not in the same region as the bucket the bucket's region
         # this is a temporary fix, after https://github.com/scylladb/mermaid/issues/1456 is fixed, this is not necessary
-        cred_file = '/etc/scylla-manager-agent/scylla-manager-agent.yaml'
+        config_file = '/etc/scylla-manager-agent/scylla-manager-agent.yaml'
         region = 'us-east-1'
```

Comment:
Why is the region hard coded and not taken from the node region?

Reply:
because this region is related with the bucket's region, not with the node's region

Reply:
We should have bucket per region, configured in aws_config.yaml per region, the backup should use it automatically.

Another approach is to create and configure the backup on the fly.

In any case, nothing should be hard coded like that.

Reply:
it now comes from the test_default.yaml